index.html

```diff
@@ -10,6 +10,21 @@
 <body>
   <header>
     <a href="#" id="my-logo">Farida's Logo</a>
+    <img src="images/home.png" alt="description" id="home"/>
   </header>
-      </body>
+
+  <section id="intro">
+    <section id="text">
+      <h1>Farida Hamid</h1>
+      <p>Software Engineer and consultant to educational platform since March 2020.</p>
+      <h3>let's connect</h3>
+      <ul id="contacts">
+        <li><a href="#"><img src="images/vec.png" alt="twitter"></a></li>
+        <li><a href="#"><img src="images/Linkedin-icon.png" alt="Linkedin-icon"></a></li>
+        <li><a href="#"><img src="images/vec-3.png" alt="email"></a></li>
+        <li><a href="#"><img src="images/vec-2.png" alt="github"></a></li>
```

Comment:
- [ Optional]  @Farida-Hamid the last code reviewer show you that image with no icons and background the reason for this is that you using local links to your images please use the relative directory to make sure that the images are working fine 
- example
![image](https://user-images.githubusercontent.com/74744787/159123058-4c937d93-8ef9-4615-a114-dddbecba0274.png)